console/docs/custom-function-call-message.md

```
@@ -0,0 +1,620 @@
+  WebToolView.tryRenderPreview(message)
+
+const customTerminal = !pending
+  ? (SandboxToolView.tryRender(message) ??
+    EngineToolView.tryRender(message) ??
+    DirectoryToolView.tryRender(message) ??
+    WorkerToolView.tryRender(message) ??
+    WebToolView.tryRender(message))
+  : null
+```
+
+> **Note on scale.** Five families chained by hand is fine; it is explicit and tree-shakes cleanly. There is no plugin *registry* yet — see [Scale beyond hand-wiring](#11-scale-beyond-hand-wiring) for the refactor sketch if the chain grows unwieldy. Until then, follow the [minimal wiring](#7-wire-the-family-into-functioncallmessagetsx) below.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)